src/components/Layout.js

```diff
@@ -58,7 +58,7 @@ export default class Template extends React.Component {
             paddingTop: 0,
           }}
         >
-          {this.props.children()}
+          {this.props.children}
```

Comment:
No longer a function, and no longer a "magic" component. It's just a regular React component now!